### DISCUSSION TITLE
Laravel Envoy - Started Compiler

### DISCUSSION THREAD

**Discussion by amandiobm** [laravel/framework#32931] [General] [answered]
Hey guys, I want to send a notification always my envoy runs. Reading the docs, it only has the `@finished` directive/compiler.

```
@finished
    @slack('webhook-url', '#bots')
@endfinished
```

I'm looking for something similar.

```
@started
    @slack('webhook-url', '#bots')
@endstarted
```

There is anything liked?!

**Comment by royduin** [comment]
Doublepost: https://github.com/laravel/framework/discussions/32930

**Comment by amandiobm** [comment, marked as answer]
Yes, sorry!
